client/src/pages/NewMates/NewMatesPage.test.tsx

```diff
@@ -48,28 +61,96 @@ test("add and remove a mate card", async () => {
       expect(screen.getByLabelText("Last Seen")).toBeEmptyDOMElement()
     );
     await waitFor(() =>
-      expect(screen.getByText("Jonathan Edwards")).toBeInTheDocument()
+      expect(screen.getByText(jonathan.name)).toBeInTheDocument()
     );
   });
 
-  // removing Alice's mate card
+  // removing Edwards' mate card
```

Comment:
jonathan's card? Is edward his last name? lol

Reply:
yeah, confusing name :/